common/json/impl/boost/boost_json.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include <common/json/impl/boost/boost_json.hpp>
+#include <boost/json/src.hpp>
+
+
+void json::BoostJson::hello_world()
```

Comment:
It is perhaps easier to put most of the file inside a `namespace json { }`, or? This also avoids leaking into the unnamed namespace if you forget it somewhere.